<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>610</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>399</y>
     <width>88</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="bpm_min_label">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>479</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>460</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Bpm min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tmp_avg_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>529</y>
     <width>92</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bpm_avg_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>479</y>
     <width>92</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tmp_max_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>529</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>510</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>tmp min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tmp_min_label">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>529</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>399</y>
     <width>88</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Day</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_data_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>460</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Bpm max</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>510</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> tmp avg</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bpm_max_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>479</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>399</y>
     <width>88</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Month</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>460</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Bpm avg</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>tmp max</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="year_combo">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>420</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="month_combo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="day_combo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>560</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="tmp_plot" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>530</width>
     <height>180</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="bpm_plot" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>530</width>
     <height>180</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Hour</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hour_combo">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="draw_data_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>510</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Data</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
